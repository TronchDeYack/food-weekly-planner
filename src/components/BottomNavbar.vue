<template>
  <v-bottom-navigation :fixed="true">
    <v-btn value="menu" to="/">
      <span>Menu</span>
      <v-icon>mdi-chef-hat</v-icon>
    </v-btn>

    <v-btn value="recipes" to="/recipes">
      <span>Recipes</span>
      <v-icon>mdi-noodles</v-icon>
    </v-btn>

    <v-btn value="shopping">
      <span>Shopping</span>
      <v-icon>mdi-cart</v-icon>
    </v-btn>
  </v-bottom-navigation>
</template>
