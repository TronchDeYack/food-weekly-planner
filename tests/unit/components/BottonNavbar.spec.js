import { shallowMount } from '@vue/test-utils';

import BottomNavbar from '@/components/BottomNavbar.vue';

describe('BottomNavbar', () => {
  it('Should render.', () => {
    const component = shallowMount(BottomNavbar);
    expect(component).toMatchSnapshot();
  });
});
