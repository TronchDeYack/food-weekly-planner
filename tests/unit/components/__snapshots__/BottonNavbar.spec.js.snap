// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomNavbar Should render. 1`] = `
<v-container-stub tag="div" class="mt-3">
  <v-bottom-navigation-stub fixed="true" height="56" inputvalue="true" activeclass="v-btn--active">
    <v-btn-stub tag="button" activeclass="" to="/" type="button" value="menu"><span>Menu</span>
      <v-icon-stub>mdi-chef-hat</v-icon-stub>
    </v-btn-stub>
    <v-btn-stub tag="button" activeclass="" to="/recipes" type="button" value="recipes"><span>Recipes</span>
      <v-icon-stub>mdi-noodles</v-icon-stub>
    </v-btn-stub>
    <v-btn-stub tag="button" activeclass="" to="/shopping" type="button" value="shopping"><span>Shopping</span>
      <v-icon-stub>mdi-cart</v-icon-stub>
    </v-btn-stub>
  </v-bottom-navigation-stub>
</v-container-stub>
`;
